packages/cascara/src/ui/Table/Table.test.snap

```diff
@@ -108,8 +108,10 @@ exports[`Table component tree snapshot test 1`] = `
             >
               <span>
                 <span
+                  aria-label="Email"
                   class="Input"
                   data-testid="email"
+                  label="Email"
```

Comment:
@lordpixel I think this is incorrect. `label` is not a valid parameter on a `span` tag.

Reply:
done, snapshots have been updated